projects/storefrontlib/cms-components/cart/cart-details/cart-details.component.html

```diff
@@ -1,9 +1,9 @@
 <ng-container *ngIf="cart$ | async as cart">
   <ng-container *ngIf="entries$ | async as entries">
     <div *ngIf="cart.totalItems > 0" class="cart-details-wrapper">
-      <h4 class="cx-total">
+      <span class="cx-total">
```

Comment:
Please add info about this change into 5.0 migration doc.

Reply:
Added